src/unix/fs.c

```diff
@@ -56,11 +56,11 @@
 #include <sys/time.h>
 #if !defined(__TIZENRT__)
 # include <sys/uio.h> /* writev */
+#include <utime.h>
```

Comment:
There are some spacing inconsistencies with the includes (here and in core.c for sure). Either the `# include` or the `#include` format should be used but not both. (I'm for the no-spce version, but that's mostly personal preference.)